packages/custom-elements/src/CustomElementRegistry.ts

```diff
@@ -79,14 +79,14 @@ export default class CustomElementRegistry {
     }
   }
 
-  define(localName: string, constructor: ElementConstructor) {
+  define(localName: string, constructor: Function) {
```

Comment:
Is this change (type previously `ElementConstructor`) related to the cast-through-`unknown` bit at the bottom? (Same question for `{new(): HTMLElement}` used in `get`.)

Reply:
No, it was just to better align the type with the type of `CustomElementsRegistry` from lib.dom.d.ts